app/code/Magento/Analytics/ReportXml/QueryFactory.php

```diff
@@ -45,36 +49,44 @@ class QueryFactory
      */
     private $selectHydrator;
 
+    /**
+     * @var Json|null
+     */
+    private $jsonSerializer;
+
     /**
      * QueryFactory constructor.
      *
-     * @param CacheInterface $queryCache
-     * @param SelectHydrator $selectHydrator
+     * @param CacheInterface         $queryCache
```

Comment:
Please revert formatting change for all arguments, like this
```suggestion
     * @param CacheInterface $queryCache
```
Please do it for all changes methods